docs/UserGuide.md

```diff
@@ -218,6 +215,7 @@ Copy student emails to the clipboard.
 Format: `copy [t/TAG]…`
 
 * When no tags are supplied, copies all student emails.
+* When multiple tags are used, copies emails of all students that has **ALL** the tags specified.
```

Comment:
Maybe add an example so it can't be said that its not clear.

* Example: `copy t/tutorial-1 t/weak` only copies emails of all students that has **BOTH** `tutorial-1` and `weak` tags.

Reply:
Example already shown below so I think ill leave it as it is.